api.js

```diff
@@ -259,6 +287,22 @@ module.exports = async function attachAPI(app, {wss, db}) {
       })
     ),
 
+    runIfCondition: (conditionFn, runIfSo, runIfNot = []) => (
```

Comment:
This middleware stuff is becoming its own DSL.

Reply:
To be fair, yes 🎉